src/View/mainpage/DrawROIWindow/UIDrawROIWindow.py

```diff
@@ -294,6 +297,39 @@ def init_layout(self):
             addRow(self.draw_roi_window_viewport_zoom_box)
 
         self.init_cursor_radius_change_box()
+
+        # Create slider to adjust the transparency of drawn pixels
+        self.transparency_slider_box = QHBoxLayout()
+        self.transparency_slider_label = QLabel()
+        self.transparency_slider_label.setObjectName("TransparencySliderLabel")
+
+        self.transparency_slider_input_box = QLineEdit()
+        self.transparency_slider_input_box.setObjectName("TransparencySliderInputBox")
+        self.transparency_slider_input_box.setText("{:.0f}".format(self.pixel_transparency * 100) + "%")
+        self.transparency_slider_input_box.setCursorPosition(0)
+        self.transparency_slider_input_box.setEnabled(False)
+        self.transparency_slider_input_box.setSizePolicy(QSizePolicy.Minimum, QSizePolicy.Minimum)
+        self.transparency_slider_input_box.resize(self.transparency_slider_input_box.sizeHint().width(),
+                                                  self.transparency_slider_input_box.sizeHint().height())
+
+        self.transparency_slider = QSlider(Qt.Horizontal)
+        self.transparency_slider.setMinimum(0)
+        self.transparency_slider.setMaximum(100)
+        self.transparency_slider.setSingleStep(1)
+        self.transparency_slider.setTickPosition(QSlider.TicksBothSides)
+        self.transparency_slider.setTickInterval(10)
+        self.transparency_slider.setValue(50)
+        self.transparency_slider.setObjectName("TransparencySlider")
+        self.transparency_slider.resize(self.transparency_slider.sizeHint().width(),
+                                        self.transparency_slider.sizeHint().height())
+        self.transparency_slider.valueChanged.connect(self.transparency_slider_value_changed)
+
+        self.transparency_slider_box.addWidget(self.transparency_slider_label)
+        self.transparency_slider_box.addWidget(self.transparency_slider_input_box)
+        self.transparency_slider_box.addWidget(self.transparency_slider)
+        self.draw_roi_window_input_container_box.addRow(self.transparency_slider_box)
+
+
```

Comment:
v picky, stick to single line spacing. No need to change

Reply:
fixed